content/project-organization/quality-council/processes/issue-management.md

```diff
@@ -409,37 +418,35 @@ We only use GitHub issues to discuss about bugs and new features in the PrestaSh
 Thank you
 ```
 
-#### Issue is too technical to reproduce
+### Issue is too technical to reproduce
 
 ```
 Hello,
 
 Thanks for reporting this issue!
-Ping @PrestaShop/prestashop-core-developers: Could someone please try to reproduce the issue, it's too technical!
+Ping @PrestaShop/committers : Could someone please try to reproduce the issue, it's too technical!
 
 Thanks in advance!
 ```
 
-#### Issue about an archived module
+### Issue about an archived module
 
-```text
+```
 Hello,
 
-This module has been archived, it's no longer maintained but you still can fork it if you need it for your shop.
-
-Also let us know if you are willing to maintain this module.
+This module has been archived, it’s no longer maintained but you still can fork it if you need it for your shop.
+Also let us know if you are voluntary to maintain this module.
 
 Thanks in advance!
 ```
 
 ## Miscellaneous notes (to be read!)
 
-1. The `NMI` (Need More Info) tag is used at the beginning of the life of the issue, when there are still questions about the issue (quality, completeness, etc).
-2. `Waiting for Author` is used for conversations with the author and for `NMI` issues when we answer. The **issue-bot** will delete the label when the author answers (to help us filter the issues that needs answers, check the [filter](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3ANMI+-label%3A%22Waiting+for+author%22)).
-3. In case of regression discovery, it is essential to share the regressions on #dev-core (Public Slack) and to put them in the right project (e.g. if the regression is on PS 8.0.0, we put it in the project 8.0.x, then the PM will prioritize it).
-4. Don't hesitate to ping the devs/product/QA on issues where their expertise could help, and to add the corresponding label (`Waiting for PM`, `Waiting for Dev`, etc).
+1. The "NMI" (Need More Info) tag is used at the beginning of the life of the issue, when there are still questions about the issue (quality, completeness, etc). 
+2. "Waiting for Author" is used for conversations with the author and for NMI issues when we answers. The issue-bot will delete the label when the author answers (to help us filtering the issues that needs answers, check the [filter](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3ANMI+-label%3A%22Waiting+for+author%22)).
+3. In case of regression discovery, it is essential to share the regressions on #qa-core (PS Slack) and #dev-core (Public Slack) and to put them in the right project (e.g. if the regression is on PS 1.7.8.5, we put it in the project 1.7.8.5 and in the project [Maxi Kanba](https://github.com/orgs/PrestaShop/projects/7)n, then the PM will prioritize it).
```

Comment:
```suggestion
3. In case of regression discovery, it is essential to share the regressions on #dev-core (Public Slack) and to put them in the right project (e.g. if the regression is on PS 1.7.8.5, we put it in the project 1.7.8.5 and in the project [Maxi Kanba](https://github.com/orgs/PrestaShop/projects/7)n, then the PM will prioritize it).
```